.travis.yml

```diff
@@ -1,10 +1,17 @@
 language: ruby
+
+sudo: false
```

Comment:
This just opt into using Travis's new infrastructure btw, which is faster. There's no way you need sudo  access to run your test suite, so no downside.